### DISCUSSION TITLE
Dashboard to view as stand alone like chart.

### DISCUSSION THREAD

**Discussion by queppelin-dev** [apache/superset#14343] [Q&A / Help]
Hi all,
 How can I show Dashboard as standard alone in an iframe without a Header? 
For the Chart i use URL/superset/explore/?r=165&standalone=true.
![withheader](https://user-images.githubusercontent.com/56347200/116030567-72ab1380-a679-11eb-821b-2ef0fccc7391.png)
![without header](https://user-images.githubusercontent.com/56347200/116030571-7474d700-a679-11eb-9cbc-30ff9a08b9c6.png)

Please check the files and let me know if I can achieve this by any idea.
wherein image 1 is with a header which I have to remove like image 2.
Thanks

**Comment by nytai** [comment]
Does appending `?standalone=true` to the dashboard url not work?

**Reply by queppelin-dev** [reply]
url/r/214?standalone=true I am using like this and it is showing header also.

**Reply by queppelin-dev** [reply]
I need to use this to the share URL.

**Reply by villebro** [reply]
Try `?standalone=2`, I believe that removes some more headers.

**Reply by queppelin-dev** [reply]
no, it is not working, hey @villebro  can please give me an example with a complete URL.

**Reply by queppelin-dev** [reply]
hey, @nytai I need to show this dashboard on iframe.

**Reply by etr2460** [reply]
Re the conversation in slack, I think this is an issue only with the shared short URL. cc @bkyryliuk who has a fix for this that he might PR back into open source

**Reply by nytai** [reply]
@queppelin-dev instead of using the share link try `/superset/dashboard/:id?standalone=true` or append `?standalone=true` to the dashboard url when you're viewing it in a browser

**Reply by queppelin-dev** [reply]
what parameters are added when we click on toggle screen ?
